<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWindow</class>
 <widget class="QWidget" name="ConnectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>New Connection</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>25</number>
   </property>
   <item>
    <widget class="QLabel" name="dbTypeLabel">
     <property name="text">
      <string>Database Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="dbTypeComboBox">
     <item>
      <property name="text">
       <string>SQLite</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="hostNameLabel">
     <property name="text">
      <string>Host Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="hostNameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="dbNameLabel">
     <property name="text">
      <string>Database Name</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dbNameLayout">
     <item>
      <widget class="QLineEdit" name="dbNamelineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="userNameLabel">
     <property name="text">
      <string>User Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="userNameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordLineEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ConnectPushButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClosePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
